app/tests/spec/views/complete_reset_password.js

```diff
@@ -361,9 +364,12 @@ function (chai, sinon, p, AuthErrors, Metrics, FxaClient, InterTabChannel,
                   EMAIL,
                   PASSWORD,
                   relier,
+                  // TODO: remove once https://github.com/eslint/eslint/issues/1801 gets fixed
+                  /* eslint-disable */
```

Comment:
What we currently have looks awkward imo, so I don't mind changing it for the linter.